server_installation/topics/operator/external-database.adoc

```diff
@@ -40,7 +39,7 @@ The other properties work in the same way for a hosted or external database. Set
 * `POSTGRES_PASSWORD` - Database password
 * `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
```

Comment:
We should remove this.

Reply:
I believe the operator upgrading guide needs some modifications:
> The operator removes all Red Hat Single Sign-On replicas, **_optionally creates a backup_** and then creates the replicas based on a newer Red Hat Single Sign-On image.

> This ensures a zero-downtime upgrade but is more suitable for minor version upgrades that do not require database migration since the database is accessed by multiple Red Hat Single Sign-On versions concurrently. **_Automatic backups are not supported with this strategy._**